web/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -58,14 +58,18 @@
                     <div id="stream_settings_title" class="stream-info-title">{{t 'Channel settings' }}</div>
                 </div>
                 <div class="nothing-selected">
+                    <button type="button" class="create_stream_button animated-purple-button" {{#unless can_create_streams}}disabled{{/unless}}>{{t 'Create channel' }}</button>
                     {{#if can_create_streams}}
-                    <button type="button" class="create_stream_button animated-purple-button">{{t 'Create channel' }}</button>
-                    <span class="settings-empty-option-text">
+                    <span>
                         {{#tr}}
                             First time? Read our <z-link>guidelines</z-link> for creating and naming channels.
                             {{#*inline "z-link"}}<a href="/help/getting-your-organization-started-with-zulip#create-channels" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
                         {{/tr}}
                     </span>
+                    {{else}}
+                    {{#tr}}
+                        You do not have permission to create channels in this organization.
```

Comment:
You can just directly use `{{t "You do not ..." }}` instead of using `tr` tag here, since there are no variables in the string.